modules/core/src/main/java/org/apache/ignite/internal/hlc/HybridTimestamp.java

```diff
@@ -121,6 +122,15 @@ public static long hybridTimestampToLong(@Nullable HybridTimestamp timestamp) {
         return timestamp == null ? NULL_HYBRID_TIMESTAMP : timestamp.time;
     }
 
+    /**
+     * Restores a timestamp converted to bytes using {@link #toBytes()}.
```

Comment:
Perhaps you wanted to indicate a different method?

Reply:
No, I actually meant `HybridTimestamp#toBytes()`

Reply:
Ok.